Content.Client/UserInterface/Systems/Character/CharacterUIController.cs

```diff
@@ -121,11 +123,17 @@ private void CharacterUpdated(CharacterData data)
                 Modulate = Color.Gray
             };
 
-            objectiveControl.AddChild(new Label
+
+            var objectiveText = new FormattedMessage();
+            objectiveText.TryAddMarkup(Loc.GetString($"objective-issuer-{groupId}"), out _);
```

Comment:
i think this type of locale support is very bad, at least because its hardcoded
probably ObjectiveComponent should has like
```
  - type: Objective
    issuer: issuer-name-spiderclan
```
and in this part of code it should take the locale from component
this type of locale related to #21525 , because there is the same thing
here is the example of code
https://github.com/space-wizards/space-station-14/blob/061ec432ffa9d89231b52dbe39573843e6bb995c/Content.Client/Communications/UI/CommunicationsConsoleMenu.xaml.cs#L80

Reply:
I know, it's a terrible kind of localization, I really don't like it. But cutting it out completely here and replacing it with an adevate implementation would require more work, which I'm not ready to do yet. This is just a patch to remove a bug in the UI